<launch>
	<node pkg="astar_alpha" type="main.py" name="astar_main" output="screen">
	      <param name="corner1x" type="double" value="-6.25"/>
	      <param name="corner1y" type="double" value="8.2"/>
	      <param name="corner2x" type="double" value="15.75"/>
	      <param name="corner2y" type="double" value="28.2"/>
	      <param name="numCells" type="int" value="50"/>
	</node>
</launch>